Automattic/jetpack
# Social: Add media conversion settings endpoints

Fixes 1205142973434174-as-1205074394227482

## Proposed changes:

* Added endpoints to get/update auto conversion settings.

### Other information:


## Jetpack product discussion


1205142973434174-as-1205074394227482

## Does this pull request change what data or activity we track or use?


No
## Testing instructions:






* cd projects/packages/publicize
* composer run test-php
* Sanity check: Boot a JN site up on this branch and make sure publicize works correctly and updating SIG settings continue to work correctly.
